starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+
+
+const StyledName = styled.h1`
+font-size: 50px;
+color: red;
+`
+
+const StyledContainer = styled.div`
+border: 2px solid black;
+margin: 2rem 0 2rem 0;
+`
```

Comment:
For Styled Components typically you would indent your styles like so
```js
const StyledContainer = styled.div`
    border: 2px solid black;
    margin: 2rem 0 2rem 0;
`
```

This makes it look more like CSS and typically easier to read.